cmd/kyma/init/function/function_test.go

```diff
@@ -52,13 +52,13 @@ func TestFunctionFlags(t *testing.T) {
 
 	err = c.ParseFlags([]string{
 		"-d", "/tmpfile",
-		"-r", "nodejs18",
+		"-r", "nodejs20",
```

Comment:
IMO in this place better use non default runtime. if we have bug and ignore `-r` flag, result has default runtime and test is useless.